tests/baselines/reference/declFileConstructors.js

```diff
@@ -248,7 +248,7 @@ export declare class ConstructorWithPrivateParameterProperty {
     constructor(x: string);
 }
 export declare class ConstructorWithOptionalParameterProperty {
-    x?: string;
+    x?: string | undefined;

```

Comment:
Hm, I think we'd still rather this not add `undefined`, since the original declaration didn't have `undefined`, for the sake of `exactOptionalPropertyTypes`. Maybe we need to consider that flag's logic during serialization as well?

Reply:
To my surprise, the implementation was simpler than I expected.